README.md

```diff
@@ -4,9 +4,9 @@
 All software for the 2018 UWRT Mars Rover
 
 
-To get all required dependencies, navigate to the project root directory and run: 
+To get all required dependencies, navigate to the project Workspace directory and run: 
```

Comment:
can you put the actual `cd` command

Reply:
That would depend on where the project is located no? Unless you mean `cd` from root?